extension/chrome/settings/inbox/inbox.ts

```diff
@@ -169,6 +169,9 @@ export class InboxView extends View {
     BrowserMsg.addListener('show_attachment_preview', async ({ iframeUrl }: Bm.ShowAttachmentPreview) => {
       await Ui.modal.attachmentPreview(iframeUrl);
     });
+    BrowserMsg.addListener('show_confirmation', async ({ message }: Bm.ShowConfirmation): Promise<Bm.Res.ShowConfirmationResult> => {
+      return Ui.modal.confirm(message);
```

Comment:
`return await Ui.modal.confirm(message)`,
since this function is declared as `async`

Reply:
right - thank you.